src/ctelnet.cpp

```diff
@@ -120,6 +120,8 @@ cTelnet::cTelnet(Host* pH)
     connect(&socket, SIGNAL(disconnected()), this, SLOT(handle_socket_signal_disconnected()));
     connect(&socket, SIGNAL(readyRead()), this, SLOT(handle_socket_signal_readyRead()));
 
+    socket.setSocketOption(QAbstractSocket::KeepAliveOption, true);
```

Comment:
The documentation for these:
> `enum QAbstractSocket::SocketOption`
> This enum represents the options that can be set on a socket. If desired, they can be set after having received the `connected()` signal from the socket or after having received a new socket from a `QTcpServer`.

So this is being done in the second way "after having received a new socket from a `QTcpServer`", right?  Or should it be done in the `handle_socket_signal_connected()` slot when we actually have a connection to "*keep alive*"?